net/DescriptionViewController.m

```diff
@@ -86,104 +174,527 @@ -(void)issueImageDidLoad
     {
         NSLog(@"in description issue did load: ![self.issueImageView.image isEqual:cItems.issueImage], setOutlet");
         dispatch_async(dispatch_get_main_queue(), ^{
-           self.issueImageView.image = [CurrentItems sharedItems].issueImage; 
+            self.issueImageView.image = [CurrentItems sharedItems].issueImage;
         });
         
     }
 }
 
-
-
--(void)viewDidLoad
+-(void)calculateContentViewStaticHeight
 {
-    CurrentItems *cItems = [CurrentItems sharedItems];
-    [cItems.issueImageDelegates addObject:self];
-    self.navigationController.navigationBar.tintColor = [UIColor whiteColor];
-    self.navigationController.navigationBar.barTintColor = [UIColor bawlRedColor];
-    self.navigationController.navigationBar.titleTextAttributes = @{NSForegroundColorAttributeName : [UIColor whiteColor]};
-    self.navigationController.navigationBar.barStyle = UIBarStyleBlack;
+    self.contentStaticHeight = 0;
+    for (UIView *view in self.viewsVertical)
+    {
+        self.contentStaticHeight += view.frame.size.height;
+        NSLog(@"append view height: %f",view.frame.size.height);
+    }
+    for (NSLayoutConstraint *con in self.constraintsVertical)
+    {
+        self.contentStaticHeight += con.constant;
+        NSLog(@"append consthaint height: %f",con.constant);
+        
+    }
+
 }
 
+
 -(void)setDataToView
 {
-    self.titleLabel.text = [CurrentItems sharedItems].issue.name;
+    Issue *currentIssue = [CurrentItems sharedItems].issue;
+    self.titleLabel.text = currentIssue.name;
     self.titleLabel.textColor = [UIColor bawlRedColor];
-    self.descriptionLabel.text = [CurrentItems sharedItems].issue.issueDescription;
+    self.descriptionLabel.text = currentIssue.issueDescription;
+    self.currentStatusLabel.text = currentIssue.status;
     
-    NSString *firstPart = @"Issue status: ";
-    NSMutableAttributedString *aStr = [[NSMutableAttributedString alloc] initWithString:[firstPart stringByAppendingString:[CurrentItems sharedItems].issue.status]];
-    [aStr addAttribute:NSForegroundColorAttributeName value:[UIColor blueColor] range:NSMakeRange(firstPart.length, [aStr string].length - firstPart.length)];
-    self.currentStatusLabel.attributedText = aStr;
+    self.categoryImageView.image = [[IssueCategories standartCategories] imageForCurrentCategory];
```

Comment:
In class IssueCategories I have property NSArray *categories. So to get an element of array (to get the array itself) I need object (instanse of this class). So if I change this method from object method to class method I will have to call [IssueCategories standartCategories] in that method, and then get array. Isn't it the same?